### ISSUE TITLE
[BUG] Remove BrowserAnimationsModule

### ISSUE THREAD

**Issue by patryk0605** [mobi/goponents#195]
**Describe the bug**
`BrowserAnimationsModule` should only be imported once per Angular application together with `BrowserModule` because `BrowserAnimationsModule` [also exports `BrowserModule` again](https://angular.io/api/platform-browser/animations/BrowserAnimationsModule). Otherwise a user will encounter the following error: 

```
Error: Uncaught (in promise): Error: BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.
Error: BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.
```

The above error specifically comes up when attempting to lazy load modules. The recommendation per https://github.com/angular/angular-cli/issues/5684 is that `BrowserAnimationsModule` should not be imported by npm packages and instead advertised as a setup step. This is currently how Angular Material is setup within the getting started documentation: https://material.angular.io/guide/getting-started

**To Reproduce**
1. Within an angular app, run `ng generate module customers --route customer-list --module app.module` (taken from https://angular.io/guide/lazy-loading-ngmodules). 
2. Within Customers, import a goponent that uses `BrowserAnimationsModule` such as `GoLoaderModule`.
3. Attempt to navigate to Customers page and the page should not load with above mentioned error. 

**Expected behavior**
A module using goponents should be able to lazy load without crashing.

**Comment by grahamhency** [comment]
@eligolding interesting bug. Are you implementing lazy loading in Cloud currently?

**Comment by patryk0605** [comment]
@grahamhency yes, I have been implementing lazy-loading within analytics app and that's how I ran into this issue.

**Comment by grahamhency** [comment]
@patryk0605 is this something you would like to work on? There are a few places we import this I believe.

**Comment by patryk0605** [comment]
@grahamhency - sure I have no problem working on this.

**Comment by grahamhency** [comment]
@patryk0605 sounds good, I've assigned you and updated the issue.
